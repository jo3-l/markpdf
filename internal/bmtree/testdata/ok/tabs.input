p1. abc
	p2. cde