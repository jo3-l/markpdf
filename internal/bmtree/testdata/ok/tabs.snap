- "abc" @ p1
	- "cde" @ p2